support-frontend/assets/helpers/subscriptionsForms/__tests__/validationTest.ts

```diff
@@ -7,34 +7,41 @@ import {
 	notNull,
 	validate,
 } from '../validation';
+
 // ----- Tests ----- //
 describe('validation', () => {
 	describe('non silly characters', () => {
-		it('should return false if string contains a silly character', () => {
+		it('should return false if string contains an emoji', () => {
 			expect(nonSillyCharacters('😊')).toBe(false);
 			expect(nonSillyCharacters('jane✅')).toBe(false);
-			expect(nonSillyCharacters('𝒢℞à©ℨ')).toBe(false);
 		});
+
 		it('should return true if string does not contain silly characters', () => {
 			expect(nonSillyCharacters('joe')).toBe(true);
+			expect(nonSillyCharacters("King's Place")).toBe(true);
```

Comment:
```suggestion
			expect(nonSillyCharacters("King's Place")).toBe(true); // this is a special iOS apostrophe
```

Reply:
It isn't, but I've just realised there's an issue with the component test that is using a right single quotation character.

Reply:
fair enough! in that case don't use that comment :)